README.md

```diff
@@ -45,7 +45,7 @@ Before you can start development you need to create a .env.local in your root fo
 In this file please paste the following:
 
 ```
-API_BASE_URL=http://localhost:3000
+API_BASE_URL=https://wcc-backend.fly.dev/api/cms/v1
```

Comment:
Are you sending this via environment variable?

Reply:
yes, that's env variable